proteus/mprans/SpatialTools.py

```diff
@@ -86,6 +88,60 @@ def _attachAuxiliaryVariable(self, key):
                 self.auxiliaryVariables[key] = True
             if key == 'RelaxZones':
                 self.auxiliaryVariables[key] = self.zones
+            if str(key).startswith('Gauge_'):
+                self.auxiliaryVariables[key] = [gauge]
+        elif str(key).startswith('Gauge_') \
+                and gauge not in self.auxiliaryVariables[key]:
+            self.auxiliaryVariables[key] += [gauge]
```

Comment:
Great. I like that. I'd probably make the log level 0 or 1 unless this is a keys/gauges falling through is  something you expect a well-written application to generate a lot  of.